vendor/engines/saml_authentication/lib/saml_authentication/saml_authenticatable_with_custom_error.rb

```diff
@@ -21,7 +21,7 @@ class SamlAuthenticatableWithCustomError < SamlAuthenticatable
       # failure message for authentication via SAML.
       def failed_auth(msg)
         DeviseSamlAuthenticatable::Logger.send(msg)
-        fail!(:saml_invalid)
+        fail!(I18n.t("devise.failure.saml_invalid").html_safe)
```

Comment:
It's probably a little overkill here, but we could use `text-helpers`'s `html` method and then the output could be in markdown.